assets/css/style.scss

```diff
@@ -3,3 +3,11 @@
 
 @import "{{ site.theme }}";
 .container { max-width: none; width: 100%; }
+
+thead {
+    /* fix at top of screen on scroll */
+    position: sticky;
+    top: 0;
+    /* match body background */
+    background-color: #151515 url("../images/bkg.png") 0 0;
```

Comment:
@meatcar what is this image reference about?

Reply:
I'm directly copying the [`background` property from the `hacker` theme](https://github.com/pages-themes/hacker/blob/e3c8d1ad288894be216002a5bc29fc611aeab9ac/_sass/jekyll-theme-hacker.scss#L20) to make the table header opaque. Couldn't think of an SCSS way to pull that out programmatically. Updated the commit with a comment to document that.